packages/xo-server/src/api/disk.mjs

```diff
@@ -164,12 +164,12 @@ async function handleImport(req, res, { type, name, description, vmdkData, srId,
       if (part.name !== 'file') {
         promises.push(
           (async () => {
-            const view = new DataView((await getStream.buffer(part)).buffer)
-            const result = new Uint32Array(view.byteLength / 4)
-            for (const i in result) {
-              result[i] = view.getUint32(i * 4, true)
-            }
-            vmdkData[part.name] = result
+            const buffer = await getStream.buffer(part)
+            vmdkData[part.name] = new Uint32Array(
+              buffer.buffer,
+              buffer.byteOffset,
+              buffer.length / Uint32Array.BYTES_PER_ELEMENT
+            )
```

Comment:
Can you explain this change please?

Reply:
It's a copy/paste of nodeJS' documentation on the topic: https://nodejs.org/api/buffer.html#buffer_buffers_and_typedarrays

Reply:
What have you fixed in this PR?

Reply:
I don't exactly understand the question, very small VMDK used to be corrupted on import because `view.byteLength / 4 != (await getStream.buffer(part)).buffer.byteLength`. In particular because the `DataView` will want a multiple of 8 byte length to be able to perform a setBigInt64() call. When I saw the size was wrong I re-googled "how to transform a buffer in Uint32Array" and I stumbled on nodeJS documentation example and used it.